awx/main/isolated/manager.py

```diff
@@ -393,9 +436,15 @@ def run(self, instance, private_data_dir, playbook, module, module_args,
             [instance.execution_node],
             verbosity=min(5, self.instance.verbosity)
         )
+
         status, rc = self.dispatch(playbook, module, module_args)
         if status == 'successful':
             status, rc = self.check()
+
+            if self.instance.is_containerized:
+                ca_cert = self.kubectl_or_oc_vars.get('ansible_kubectl_ca_cert', None)
+                if ca_cert:
```

Comment:
This cleanup should be removed in favor of a more general cleanup that can account for failure and cleanup.